capstone/src/main/resources/static/script.js

```diff
@@ -35,94 +40,101 @@ function createEventElement(event) {
   eventElement.className = 'event';
   // Name 
   const nameElement = document.createElement('p');
-  nameElement.innerText = event.name;
-
-  const idElement = document.createElement('p');
-  idElement.innerText = event.datastoreId;
-  idElement.style.display = 'none';
-
+  nameElement.innerText = event.eventTitle;
   /*
   Time
   Location
   Organization
   Description
   */
-  eventElement.appendChild(idElement);
   eventElement.appendChild(nameElement);
   eventElement.appendChild(createReviewElement(event));
   
   return eventElement;
 }
 
 /**
- * Format review element and listing
+ * Create event's review submission and format review listing
  */
 function createReviewElement(event) {
-  const reviewElement = document.createElement('span');
+  const reviewElement = document.createElement('div');
+
+  //const reviewFormElement = document.createElement('form');
+  //reviewFormElement.setAttribute('action', newReview(event.datastoreID, reviewInputElement.value));
 
-  // Submission
   const reviewInputElement = document.createElement('input');
   reviewInputElement.setAttribute('placeholder', 'Leave a review');
   reviewInputElement.setAttribute('type', 'text');
+  //reviewInputElement.required = true;
 
+  //const reviewButtonElement = document.createElement("input"); //input element, Submit button
+  //reviewButtonElement.setAttribute('type','submit');
+  //reviewButtonElement.setAttribute('value','Submit');
+
+  //reviewFormElement.appendChild(reviewInputElement);
+  //reviewFormElement.appendChild(reviewButtonElement);
   // Future: option to add image
 
   const reviewButtonElement = document.createElement('button');
   reviewButtonElement.innerText = 'Submit Review';
   reviewButtonElement.addEventListener('click', () => {
-    newReview(event.datastoreId, reviewInputElement.value);
+    if (reviewInputElement.value != '') {
```

Comment:
Try the `required` attribute that is new in HTML 5? I'm not sure if it works in Chrome.
https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/required